tests/alive-tv/memory/mallocs-null.src.ll

```diff
@@ -1,92 +0,0 @@
-; target: 64 bits ptr addr
```

Comment:
This test was deleted because it assumed that malloc can nondeterministically return null.